doc/api/esm.md

```diff
@@ -308,6 +308,35 @@ modules it can be used to load ES modules.
 The `import.meta` meta property is an `Object` that contains the following
 properties.
 
+### `import.meta.dirname`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.2 - Release candidate
+
+* {string} The directory name of the current module. This is the same as the
+  [`path.dirname()`][] of the [`import.meta.filename`][].
+
+> **Caveat**: only present on `file:` modules.
+
+### `import.meta.filename`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.2 - Release candidate
```

Comment:
I'm not clear if I should be changing these YAML blocks, and what I should be altering them to if so.

Reply:
@jsumners as I understand, the `REPLACEME` is placeholder for a version that should be a release version, like this:

<!-- YAML
added: 1.2.0
-->

Hence, this readme file should be updated on each release with the accorded version.